### ISSUE TITLE
"NAT traffic is processed by CPU" when WAN Mac is cloned

### ISSUE THREAD

**Issue by vadimgrn** [MerlinRdev/bluecave-merlin#18]
NAT traffic is processed by CPU" when WAN Mac is cloned, "PPA (Protocol Processing Accelerator) is enabled" with cloning disabled. Advanced Settings/LAN/Switch Control/NAT Acceleration.
This bug was revealed into this thread https://www.snbforums.com/threads/asus-blue-cave-ac2600-dual-band-wireless-router-reviewed.45982/page-4, but it still exists. I hope you will fix it :)

**Comment by paldier** [comment]
PPA is disabled when you are using MAC clone? This part of the code is closed, i'm not sure where the problem is. Let's wait for the asus 386 firmware to be tested.
I'm just sure you can use MAC clone with PPA in SDK firmware.

**Comment by vadimgrn** [comment]
> PPA is disabled when you are using MAC clone?
Exactly. I wasn't able to determine the reason until today I found this info in thread on snbforums.
